### DISCUSSION TITLE
How to change language in which Brackets application is displayed

### DISCUSSION THREAD

**Discussion by obidare-folu** [brackets-cont/brackets#221] [Q&A]
How do I change the language in which the options are displayed? When installing it automatically was in the language which is on my windows system but I want it in English, thanks. I can;t find any settings bar.

**Comment by abose** [comment]
debug menu > switch language
![image](https://user-images.githubusercontent.com/5336369/188565597-61661647-ac8c-4025-8735-2872679b1c3e.png)

**Comment by charlypa** [comment]
![Brackets_fnNsSNyFTv](https://user-images.githubusercontent.com/1525027/188576112-57df4693-4404-4949-b2ee-a519b7a4cedb.gif)

**Reply by Jakobson1983** [reply]
Thanks
